lte/gateway/c/oai/tasks/sctp/sctp_primitives_server.c

```diff
@@ -120,28 +145,24 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
 static void* sctp_thread(__attribute__((unused)) void* args_p) {
   itti_mark_task_ready(TASK_SCTP);
   init_task_context(
-      TASK_SCTP, (task_id_t[]){TASK_MME_APP, TASK_S1AP}, 2, handle_message,
-      &sctp_task_zmq_ctx);
+      TASK_SCTP, (task_id_t[]){TASK_MME_APP, TASK_S1AP, TASK_NGAP}, 3,
+      handle_message, &sctp_task_zmq_ctx);
 
   zloop_start(sctp_task_zmq_ctx.event_loop);
   sctp_exit();
   return NULL;
 }
 
 int sctp_init(const mme_config_t* mme_config_p) {
-  OAILOG_DEBUG(LOG_SCTP, "Initializing SCTP task interface\n");
-
   if (init_sctpd_downlink_client(!mme_config.use_stateless) < 0) {
     OAILOG_ERROR(LOG_SCTP, "failed to init sctpd downlink client\n");
   }
 
   if (itti_create_task(TASK_SCTP, &sctp_thread, NULL) < 0) {
     OAILOG_ERROR(LOG_SCTP, "create task failed\n");
-    OAILOG_DEBUG(LOG_SCTP, "Initializing SCTP task interface: FAILED\n");
     return -1;
   }
 
-  OAILOG_DEBUG(LOG_SCTP, "Initializing SCTP task interface: DONE\n");
```

Comment:
Do these Debug logs need to be removed?

Reply:
by mistake got removed, have reverted now